[WIP] List docs to site (#2282)
jpmorganchase/salt-ds: site/docs/components/list/usage.mdx

---
# Leave the frontmatter as is
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  label: Usage
data:
  $ref: ./#/data
---

### Using the List component

#### When to use List

> - To present a set of selectable options that allow users to define choices for an onward action, such as filtering a data grid, selecting a country, or assigning a task.

Author: ```suggestion
- To present a set of selectable options that allow users to make a choice for an onward action, such as filtering a data grid, selecting a country, or assigning a task.
```

#### When not to use List

> - If the List items are intended to provide navigation, consider using stacked [navigation items](/salt/components/navigation-item) to form a vertical navigation pattern.
> - If the List is likely to contain five items or fewer, consider using a [radio button group](/salt/components/radio-button).
> - If the List is likely to contain between five and 10 items and does not need to be visible at all times, consider placing the list of items within a [Dropdown](/salt/components/dropdown).
> - If the List contains more than 10 items and screen space is limited, consider using a [Combobox](/salt/components/combobox).

Reviewer: ```suggestion
- If the List items are intended to provide navigation, consider using stacked [Navigation Items](/salt/components/navigation-item) to form a vertical navigation pattern.
- If the List is likely to contain five items or fewer, consider using a [Radio Button Group](/salt/components/radio-button).
- If the List is likely to contain between five and 10 items and does not need to be visible at all times, consider placing the list of items within a [Dropdown](/salt/components/dropdown).
- If the List contains more than 10 items and screen space is limited, consider using a [Combo Box](/salt/components/combobox).
```

> - If the List contains more than 10 items and screen space is limited, consider using a [Combo Box](/salt/components/combobox).

Author: ```suggestion
- If the List contains more than 10 items and screen space is limited, consider using a [Combo Box](/salt/components/combo-box).
```
